### PR TITLE
Added cryptocurrency examples to README.rst

### PR DESCRIPTION
**Pull request by GenusGeoff**
🎉 Hello there 🎉! Thanks for taking the time to contribute to `Tiingo-Python`!

## Summary of Changes

- What is the motivation / context for this change? Adding documentation for #340 from #520 
- Is this a bugfix or a feature? Feature
- What does this change / fix? (Link to Github Issue page if applicable) README.rst

## Checklist

- [X] Code follows the style guidelines of this project
- [N/A] Added tests for new functionality
- [X] Update `HISTORY.rst` with an entry summarizing your change
- [X] Tag a project maintainer
@hydrosquall

### CODE REVIEW

**Comment by GenusGeoff** [comment]
I think that maybe I finally got this correctly merged. Thank you for bearing with me through these struggles.

File: README.rst [hunk 1]
```diff
@@ -108,19 +109,19 @@ Now you can use ``TiingoClient`` to make your API calls. (Other parameters are a
 
   # Get definitions for fields available in the fundamentals-api, ticker is
   # optional
-  definitions = get_fundamentals_definitions('GOOGL')
+  definitions = client.get_fundamentals_definitions('GOOGL')
 
   # Get fundamentals which require daily-updated (like marketCap). A start-
   # and end-date can be passed. If omited, will get all available data.
-  fundamentals_daily = CLIENT.get_fundamentals_daily('GOOGL',
+  fundamentals_daily = client.get_fundamentals_daily('GOOGL',
```
Comment (hydrosquall) [inline README.rst:116]: Good catch!

**Review by hydrosquall — APPROVED** [review]
You did it @GenusGeoff, and you caught some unrelated typos in the readme! This is great, thank you for sticking with this. This was the most critical part of #520, as many people depend on the README to figure out how to quickly use a new library, and don't venture into other documentation pages.

The one piece that got accidentally dropped in this process was the line giving you attribution for your contribution (`HISTORY.rst`) - would you be interested in adding that back in 1 more commit? If not, I'll make a note to add this when I do a future release.

File: README.rst (hunk 1 above)
Comment (GenusGeoff) [reply README.rst:116]: You can just add it on a later release. That's no problem, at all.

Comment (hydrosquall) [reply README.rst:116]: Updated in https://github.com/hydrosquall/tiingo-python/commit/37c82d21a0d7e44b2d1616642e27141d62ab4eb3

**Pull request merged**